src/main/resources/db/changelog/changeset/0026_session_main_topic/0026_changeSet.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd">
+  <changeSet author="aalicic" id="consultantStatus">
+    <rollback>
+      <sqlFile path="db/changelog/changeset/0026_session_main_topic/session-main-topic-rollback.sql"
```

Comment:
As we already have a migration convflict with OS here could you please use version 28 instead of 26 for all migration related files?